src/components/DeleteButton.jsx

```diff
@@ -0,0 +1,17 @@
+import { deleteComment } from "../../api"
+
+const DeleteButton = ({comment, comments, setComments }) => {
+    const commentsWithoutDeletedComment = comments.filter((commentItem) => {
+        return commentItem.comment_id !== comment.comment_id
+    })
+
+    const handleClick = () => {
+        setComments(commentsWithoutDeletedComment)
+        deleteComment(comment.comment_id)        
```

Comment:
We should give visual feedback to the user for a successful and an unsuccessful delete 🙂